packages/web/src/scss/components/Accordion/README.md

```diff
@@ -0,0 +1,90 @@
+# Accordion
+
+Accordion is a wrapper for multiple **Collapse** items, and can be open one or more at time.
+
+## Usage
+
+### Default (Always open)
+
+```html
+<section class="Accordion" data-toggle="accordion">
+  <article id="accordionExample_article_0" class="Accordion__item">
+    <h3 id="accordionExample_article_0_header" class="Accordion__itemHeader">
+      <button
+        type="button"
+        class="Accordion__itemToggle"
+        data-toggle="collapse"
+        data-target="accordionExample_article_0_collapse"
+      >
+        toggle
+      </button>
+      <span class="Accordion__itemSide">
+        <span class="Accordion__itemSlot"> slot </span>
+        <span class="Accordion__itemIcon">
+          <svg width="24" height="24" aria-hidden="true">
+            <use xlink:href="/icons/svg/sprite.svg#chevron-down" />
+          </svg>
+        </span>
+      </span>
+    </h3>
+    <div id="accordionExample_article_0_collapse" class="Collapse" aria-labelledby="accordionExample_article_0_header">
+      <div class="Collapse__content">
+        <div class="Accordion__content">content</div>
+      </div>
+    </div>
+  </article>
+  ...
+</section>
+```
+
+### Open only one at time
+
+```html
+<section id="accordionExample" class="Accordion" data-toggle="accordion">
+  <article id="accordionExample_article_0" class="Accordion__item">
+    <h3 id="accordionExample_article_0_header" class="Accordion__itemHeader">
+      <button
+        type="button"
+        class="Accordion__itemToggle"
+        data-toggle="collapse"
+        data-target="accordionExample_article_0_collapse"
+      >
+        toggle
+      </button>
+      <span class="Accordion__itemSide">
+        <span class="Accordion__itemSlot"> slot </span>
+        <span class="Accordion__itemIcon">
+          <svg width="24" height="24" aria-hidden="true">
+            <use xlink:href="/icons/svg/sprite.svg#chevron-down" />
+          </svg>
+        </span>
+      </span>
+    </h3>
+    <div
+      id="accordionExample_article_0_collapse"
+      class="Collapse"
+      aria-labelledby="accordionExample_article_0_header"
+      data-parent="#accordionExample"
+    >
+      <div class="Collapse__content">
+        <div class="Accordion__content">content</div>
+      </div>
+    </div>
+  </article>
+  ...
+</section>
+```
+
+## Attributes
+
+Attributes are inherited from [**Collapse** component][collapse].
+
+| Prop name     | Type     | Default | Required | Description                                                         |
+| ------------- | -------- | ------- | -------- | ------------------------------------------------------------------- |
+| `data-parent` | `string` | -       | no       | A parent element selector that ensures that only one item is opened |
```

Comment:
I think this API table can be deleted because `data-parent` belong to the `Collapse` component, not `Accordion`. Is that right?

Reply:
Ok, I only added it here because this parameter is directly related to Accordion.